src/Core/Silk.NET.Core.Win32Extras/Windows.gen.cs

```diff
@@ -18720,41 +18720,21 @@ public unsafe partial class Windows : NativeAPI
         [NativeApi(EntryPoint = "EnableTraceEx")]
         public unsafe partial uint EnableTraceEx([Flow(Silk.NET.Core.Native.FlowDirection.In)] Guid* ProviderId, [Flow(Silk.NET.Core.Native.FlowDirection.In)] Guid* SourceId, ulong TraceHandle, uint IsEnabled, byte Level, ulong MatchAnyKeyword, ulong MatchAllKeyword, uint EnableProperty, EventFilterDescriptor* EnableFilterDesc);
 
-        /// <summary>To be documented.</summary>
-        [NativeName("Src", "Line 1946, Column 1 in evntrace.h")]
-        [NativeApi(EntryPoint = "EnableTraceEx")]
-        public unsafe partial uint EnableTraceEx([Flow(Silk.NET.Core.Native.FlowDirection.In)] Guid* ProviderId, [Flow(Silk.NET.Core.Native.FlowDirection.In)] Guid* SourceId, ulong TraceHandle, uint IsEnabled, byte Level, ulong MatchAnyKeyword, ulong MatchAllKeyword, uint EnableProperty, ref EventFilterDescriptor EnableFilterDesc);
-
         /// <summary>To be documented.</summary>
         [NativeName("Src", "Line 1946, Column 1 in evntrace.h")]
         [NativeApi(EntryPoint = "EnableTraceEx")]
         public unsafe partial uint EnableTraceEx([Flow(Silk.NET.Core.Native.FlowDirection.In)] Guid* ProviderId, [Flow(Silk.NET.Core.Native.FlowDirection.In)] in Guid SourceId, ulong TraceHandle, uint IsEnabled, byte Level, ulong MatchAnyKeyword, ulong MatchAllKeyword, uint EnableProperty, EventFilterDescriptor* EnableFilterDesc);
 
-        /// <summary>To be documented.</summary>
-        [NativeName("Src", "Line 1946, Column 1 in evntrace.h")]
-        [NativeApi(EntryPoint = "EnableTraceEx")]
-        public unsafe partial uint EnableTraceEx([Flow(Silk.NET.Core.Native.FlowDirection.In)] Guid* ProviderId, [Flow(Silk.NET.Core.Native.FlowDirection.In)] in Guid SourceId, ulong TraceHandle, uint IsEnabled, byte Level, ulong MatchAnyKeyword, ulong MatchAllKeyword, uint EnableProperty, ref EventFilterDescriptor EnableFilterDesc);
-
         /// <summary>To be documented.</summary>
         [NativeName("Src", "Line 1946, Column 1 in evntrace.h")]
         [NativeApi(EntryPoint = "EnableTraceEx")]
         public unsafe partial uint EnableTraceEx([Flow(Silk.NET.Core.Native.FlowDirection.In)] in Guid ProviderId, [Flow(Silk.NET.Core.Native.FlowDirection.In)] Guid* SourceId, ulong TraceHandle, uint IsEnabled, byte Level, ulong MatchAnyKeyword, ulong MatchAllKeyword, uint EnableProperty, EventFilterDescriptor* EnableFilterDesc);
 
-        /// <summary>To be documented.</summary>
-        [NativeName("Src", "Line 1946, Column 1 in evntrace.h")]
-        [NativeApi(EntryPoint = "EnableTraceEx")]
-        public unsafe partial uint EnableTraceEx([Flow(Silk.NET.Core.Native.FlowDirection.In)] in Guid ProviderId, [Flow(Silk.NET.Core.Native.FlowDirection.In)] Guid* SourceId, ulong TraceHandle, uint IsEnabled, byte Level, ulong MatchAnyKeyword, ulong MatchAllKeyword, uint EnableProperty, ref EventFilterDescriptor EnableFilterDesc);
```

Comment:
This is invalid, we are missing bindings to evntprov.h.

Reply:
Even adding eventprov.h it still is missing?

Reply:
It's in shared, not um.

Reply:
nah it was just the include order, it is in um

Reply:
Not on my machine? C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um\evntprov.h does not exist